luigi/interface.py

```diff
@@ -38,31 +36,7 @@
 from luigi import worker
 from luigi import execution_summary
 from luigi.cmdline_parser import CmdlineParser
-
-
-def setup_interface_logging(conf_file='', level_name='DEBUG'):
```

Comment:
A quick question. We call this method directly to set up logging. What is the equivalence now? Or it is not needed anymore?